guides/v2.1/migration/extend-the-tool.md

```diff
@@ -119,7 +119,7 @@ Steps can include four types of classes:
 - Delta delivering
 
 {: .bs-callout .bs-callout-info }
-Refer to [Configuration]({{ page.baseurl }}/migration/migration-tool-internal-spec.html#configuration), [Step internals]({{ page.baseurl }}/migration/migration-tool-internal-spec.html#step-internals), [Stages]({{ page.baseurl }}/migration/migration-tool-internal-spec.html#stages) and [Running modes]({{ page.baseurl }}/migration/migration-tool-internal-spec.html#running-modes) for more information.
+Refer to [Configuration]({{ page.baseurl }}/migration/migration-tool-internal-spec.html#configuration), [Step internals]({{ page.baseurl }}/migration/migration-tool-internal-spec.html#step-internals), [Stages]({{ page.baseurl }}/migration/migration-tool-internal-spec.html#step-stages) and [Running modes]({{ page.baseurl }}/migration/migration-tool-internal-spec.html#running-modes) for more information.
```

Comment:
Could you check the link once again, please? `/migration/migration-tool-internal-spec.html#stages` works correctly but `/migration/migration-tool-internal-spec.html#step-stages` does not since there's no such header in the document.

Reply:
@rogyar Stages header was renamed to Step Stages in this PR in order to make it clearer for the reader. I have checked all the links to make this change backward compatible. Please let me know if updates are needed.